.github/workflows/release-trigger.yaml

```diff
@@ -0,0 +1,31 @@
+name: Check Version Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Push tag for each updated package
+        run: |
+          git config --global user.name "GitHub Action"
+          git config --global user.email "github-action@github.com"
```

Comment:
For the SDK update PRs we set `"SDK Generator Bot"` and `"noreply@stackit.de"` as name and email. Maybe here we could follow the same approach, with name something like `"SDK Releaser Bot"`?

Reply:
I think that email is associated with https://github.com/stackit-pipeline